core/src/test/java/org/infinispan/partitionhandling/PreferConsistencyRestartTest.java

```diff
@@ -182,6 +186,11 @@ public void testCoordinatorChangesWhileDegraded() throws Exception {
       checkPersistentUUIDMatch(addressMappings);
    }
 
+   private void eventuallyClusterTopologyCoordinator(int index) {
+      ClusterTopologyManager ctm = TestingUtil.extractGlobalComponent(manager(index), ClusterTopologyManager.class);
+      eventually(() -> ctm.getStatus() == ClusterTopologyManager.ClusterManagerStatus.COORDINATOR);
```

Comment:
Was this an issue before? I am confused why this is happening now and why the ClusterTopologyManager would be in a different state that the manager itself as we assert it is the coordinator just a few lines above with `assertThat(manager(0).isCoordinator()).isTrue();`. Can you confirm why?